### PR TITLE
Parse TRUE and FALSE as keyword

### PR DESCRIPTION
**Pull request by acreux**
Before, 
```
SELECT true
```
transpiled to
```
SELECT "true"
```
with `identify=True`

now, it transpiles to
```
SELECT TRUE
```

The output is `TRUE` and not `true`, as both forms seem to be valid

### CODE REVIEW

File: sqlglot/expressions.py [hunk 1]
```diff
@@ -488,6 +488,14 @@ class Null(Expression):
     arg_types = {}
 
 
+class _True(Expression):
```
Comment (tobymao) [inline sqlglot/expressions.py:491]: can we change this to 

class Boolean(Expression):
   pass

File: sqlglot/generator.py [hunk 2]
```diff
@@ -450,6 +450,14 @@ def null_sql(self, expression):
         # pylint: disable=unused-argument
         return "NULL"
 
+    def _true_sql(self, expression):
```
Comment (tobymao) [inline sqlglot/generator.py:453]: will become _boolean_sql

File: sqlglot/parser.py [hunk 3]
```diff
@@ -77,6 +77,7 @@ def _parse_decimal(args):
         TokenType.DESC,
         TokenType.ENGINE,
         TokenType.EXPLAIN,
+        TokenType.FALSE,
```
Comment (tobymao) [inline sqlglot/parser.py:80]: is it legal to name a column false or true?

Comment (acreux) [reply sqlglot/parser.py:80]: yes, it is legal in Spark to name a column `true` `TRUE`

**Pull request merged**